configure

```diff
@@ -508,6 +523,10 @@ def configure_arm(o):
   o['variables']['arm_thumb'] = 0      # -marm
   o['variables']['arm_float_abi'] = arm_float_abi
 
+  if options.dest_os == 'android':
+    o['variables']['arm_fpu'] = 'vfpv3'
+    o['variables']['arm_version'] = '7'
```

Comment:
No they aren't but V8 only compiled to me for ARMv7 devices using vfpv3 or up arguments (like VFPv3-D16, VFPv3-D32, etc) are supported to compile. ARMv6 should be compiled with VFPv2 that is not supported for V8 anymore.